test/gtest/stringify.cpp

```diff
@@ -125,3 +123,100 @@ adding unique symbol
 
   EXPECT_EQ(ss.str(), stringifyText);
 }
+
+using HashStringifyTest = PrintTest;
+
+TEST_F(HashStringifyTest, Print) {
+  auto moduleText = R"wasm(
+    (module
+    (func $a
+      (block $block_a
+        (drop (i32.const 20))
+        (drop (i32.const 10))
+      )
+      (block $block_b
+        (drop (if (i32.const 0)
+          (i32.const 40)
+          (i32.const 5)
+        ))
+      )
+      (block $block_c
+        (drop (if (i32.const 1)
+          (i32.const 30)
+        ))
+      )
+      (block $block_d
+        (drop (i32.const 20))
+        (drop (i32.const 10))
+      )
+      (block $block_e
+        (drop (if (i32.const 1)
+          (i32.const 30)
+        ))
+      )
+      (block $block_f
+        (drop (if (i32.const 0)
+          (i32.const 30)
+        ))
+      )
+    )
+   )
+  )wasm";
+
+  Module wasm;
+  parseWast(wasm, moduleText);
+
+  std::stringstream ss;
+  HashStringifyWalker stringify = HashStringifyWalker();
+  stringify.walkModule(&wasm);
+
+  EXPECT_EQ(stringify.hashString.size(), 45u);
+
```

Comment:
No need to check this explicitly because we're matching the whole vector next. If the size is wrong, the next check will fail.
```suggestion
  EXPECT_EQ(stringify.hashString.size(), 45u);

```

Reply:
done